CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+## [0.8.2]
```

Comment:
I am not going to be releasing this change now, so can you put all of this in the unreleased section? And revert the changes related to version change in this PR

Reply:
okay